packages/jest-editor-support/src/Runner.js

```diff
@@ -17,16 +17,21 @@ const {EventEmitter} = require('events');
 const ProjectWorkspace = require('./ProjectWorkspace');
 const {jestChildProcessWithArgs} = require('./Process');
 
+import type {Options} from './types';
 // This class represents the running process, and
 // passes out events when it understands what data is being
 // pass sent out of the process
 module.exports = class Runner extends EventEmitter {
   debugprocess: ChildProcess;
-  workspace: ProjectWorkspace;
+  options: Options;
   outputPath: string;
+  workspace: ProjectWorkspace;
 
-  constructor(workspace: ProjectWorkspace) {
+  constructor(workspace: ProjectWorkspace, options?: Options) {
     super();
+    this.options = options || {
+      createProcess: jestChildProcessWithArgs,
+    };
```

Comment:
Actually, I would probably pull this out and do:

```
this._createProcess = …;
```

because it is not really just an option when you look at the class.

"Private" fields require an underscore, too, don't forget :)